shinywidgets/_shinywidgets.py

```diff
@@ -185,68 +191,33 @@ def _restore_state():
 # TODO: shiny should probably make this simpler
```

Comment:
I think this is referring to the fact that there should be something like a transformer 😅 

```suggestion
```